Libraries/StyleSheet/__tests__/normalizeColor-test.js

```diff
@@ -14,10 +14,10 @@ const {OS} = require('../../Utilities/Platform');
 const normalizeColor = require('../normalizeColor');
 
 it('forwards calls to @react-native/normalize-color', () => {
-  jest.resetModules().mock('@react-native/normalize-color', () => jest.fn());
-
+  jest.resetModules().mock('@react-native/normalize-color').normalizeColor =
+    () => jest.fn();
```

Comment:
Should this not be `normalizeColor = jest.fn()`?

Reply:
yes, it should be.